app/models/user.rb

```diff
@@ -0,0 +1,8 @@
+class User < ApplicationRecord
+  has_many :posts, foreign_key: 'author_id'
+  has_many :comments, foreign_key: 'author_id'
+  has_many :likes, foreign_key: 'author_id'
+  def last_3_posts
+    posts.limit(3).order(created_at: :desc)
+  end
```

Comment:
- [ ] **Optional:** Like the method you've written in Post class, the limit keyword is conventionally used at the end of your query.